lesson37/src/js/utils/authorizationcheck.js

```diff
@@ -0,0 +1,38 @@
+const findAuthorizationUser = async () => {
+  try {
+    const response = await fetch("https://mocki.io/v1/f6c5a4ee-1398-4e1b-9da5-4c4752f9c10b");
+    // false用
+    // const response = await fetch("https://mocki.io/v1/b896bb0b-3cdd-4851-acca-470dfe84de00");
+    if (!response.ok) {
+      console.error(`${response.status}:${response.statusText}`);
+      return;
+    }
+    const { data } = await response.json();
+    return data;
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+const setLoginUserToLocalStorage = (userData, userToken) => {
+  const usersData = JSON.parse(localStorage.getItem("morikenjuku")) ?? {};
+  const setUserData = {
+    username: userData.userName,
+    password: userData.password,
+    email: userData.email,
+    token: userToken
+  }
+  localStorage.setItem("morikenjuku", JSON.stringify({ [userToken]: setUserData, ...usersData }));
+}
+
+const init = async () => {
+  const userData = await findAuthorizationUser();
+  if (userData?.isAuthorization) {
+    const userToken = userData.id;
+    setLoginUserToLocalStorage(userData, userToken);
+    localStorage.setItem("token", userToken);
+    window.location.href = "loginuserpage.html";
+  }
+}
+
+init();
```

Comment:
Question:
When I press login, I log in automatically, but what if I want to enter the login information manually?
I couldn't touch the login screen because I logged in automatically.

Reply:
見て頂きありがとうございます🙇🏻‍♀️

以下の箇所のコメントアウトを解除して頂ければ
ご確認頂けるかと思います。

in utils/authorizationcheck.js
```js
// false用
// const response = await fetch("https://mocki.io/v1/b896bb0b-3cdd-4851-acca-470dfe84de00”);
```
お手数おかけしてすみません。
よろしくお願いします!

Reply:
お返事、ありがとうございます！
こちらの方法で、確認させていただきます。

Reply:
すみません、上記の方法で試したところ、下記のエラーが出ます。
お手数おかけしますが、ご確認いただけますでしょうか？
<img width="862" alt="スクリーンショット 2022-04-23 16 07 28" src="https://user-images.githubusercontent.com/99260576/164884393-f5943785-f753-4404-a718-38b7e61514bc.png">

<img width="625" alt="スクリーンショット 2022-04-23 16 10 03" src="https://user-images.githubusercontent.com/99260576/164884407-1d219d2e-d6d7-4610-9bc5-110e0729374d.png">

Reply:
@sae-github 
申し訳ありません。
リロードしてみたところ、エラーを出さなくなりました。
お騒がせしました。